src/server/api/routes/batch-default-values.router.js

```diff
@@ -37,8 +37,21 @@ const {
  *        required: true
  *      - name: stage
  *        in: query
- *        type: string
- *        required: true
+ *        description: Stage default values for a batch.
+ *        required: false
+ *        type: array
+ *        items:
+ *          type: string
+ *          enum:
+ *          - current
+ *          - storage
```

Comment:
I would move storage after harvest, the order will be as it is in DB: seeding, propagation, maturity, harvest, storage, delivery

Reply:
Fixed.